<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntegralTab</class>
 <widget class="QWidget" name="IntegralTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>588</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="integ_splitter_0">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="TextEdit" name="IntegExp">
         <property name="toolTip">
          <string>Expression</string>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Expression</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="integ_methods_label">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Lower bound of integral</string>
           </property>
           <property name="text">
            <string>Method</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SearchableComboBox" name="integ_methods_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6"/>
       </item>
       <item>
        <widget class="QPushButton" name="integ_prev">
         <property name="toolTip">
          <string>Preview the integral</string>
         </property>
         <property name="text">
          <string>Preview</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="integ_calc">
         <property name="toolTip">
          <string>Calculate the integral - Shift + Enter</string>
         </property>
         <property name="text">
          <string>Calculate</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="integ_splitter_1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="OutputWidget" name="integ_exact">
       <property name="toolTip">
        <string>Exact answer of calculation</string>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="placeholderText">
        <string>Output</string>
       </property>
      </widget>
      <widget class="OutputWidget" name="integ_approx">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Approximate answer of calculation</string>
       </property>
       <property name="placeholderText">
        <string>Approximate Answer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>caspy3/qt_assets/widgets/input.h</header>
  </customwidget>
  <customwidget>
   <class>SearchableComboBox</class>
   <extends>QComboBox</extends>
   <header>caspy3/qt_assets/widgets/searchable_combo.h</header>
  </customwidget>
  <customwidget>
   <class>OutputWidget</class>
   <extends>QTextBrowser</extends>
   <header>caspy3/qt_assets/widgets/output.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
